### PR TITLE
Added Laconicism v1.0

### PR DESCRIPTION
**Pull request by ryanrishi**

### CODE REVIEW

File: Casks/laconicism.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'laconicism' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.earslap.com/assets/laconicism/Laconicism.dmg'
+ # name 'Laconicism'
```
Comment (vitorgalvao) [inline Casks/laconicism.rb]: Don’t quote this.

Comment (vitorgalvao) [inline Casks/laconicism.rb]: Uncomment this, please.

**Comment by ryanrishi** [comment]
Got it, it was throwing an error before so I commented it out. Seems to be working now.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by ryanrishi** [comment]
Squashed.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 2aa5f24fe6ba11adf982317e63844a7fc6c6cc25 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**